cdi/src/test/java/brave/webmvc/TracingProducerTest.java

```diff
@@ -0,0 +1,153 @@
+package brave.webmvc;
+
+import static javax.servlet.http.HttpServletResponse.SC_INTERNAL_SERVER_ERROR;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import javax.enterprise.context.Dependent;
+import javax.enterprise.inject.Alternative;
+import javax.servlet.http.HttpServletResponse;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.WebApplicationException;
+import javax.ws.rs.core.MediaType;
+
+import brave.http.HttpTracing;
+import brave.jaxrs2.TracingClientFilter;
+import io.smallrye.config.inject.ConfigProducer;
+import okhttp3.mockwebserver.MockResponse;
+import okhttp3.mockwebserver.MockWebServer;
+import org.jboss.resteasy.microprofile.client.BuilderResolver;
+import org.jboss.weld.junit5.auto.AddBeanClasses;
+import org.jboss.weld.junit5.auto.EnableAlternatives;
+import org.jboss.weld.junit5.auto.EnableAutoWeld;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import zipkin2.Span;
+import zipkin2.reporter.Reporter;
+
+@EnableAutoWeld
+@AddBeanClasses({ConfigProducer.class, TracingProducer.class})
+@EnableAlternatives(TracingProducerTest.class)
+class TracingProducerTest {
+
+    private static final Logger log = Logger.getLogger(TracingProducerTest.class.getName());
+
+    private static final TestSpanReporter REPORTER = new TestSpanReporter();
+    private static final MockWebServer SERVER = new MockWebServer();
+
+    public interface TestInterface {
+        @GET
+        @Path("test")
```

Comment:
same also means same endpoints same names, ex Frontend Backend. I want to help support this, but can't justify a one off as it is confusing. Even other languages we keep the same examples, we should certainly do the same demo here.